packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -415,51 +415,6 @@ public CodeWriter WriteField(FieldDeclaration field)
             return WriteLine($";");
         }
 
-        public CodeWriter WriteParameterNullChecks(IReadOnlyCollection<Parameter> parameters)
-        {
-            foreach (Parameter parameter in parameters)
-            {
-                WriteVariableAssignmentWithNullCheck(parameter.Name, parameter);
-            }
-
-            WriteLine();
-            return this;
-        }
-
-        public void WriteVariableAssignmentWithNullCheck(string variableName, Parameter parameter)
```

Comment:
Note to reviewer: given that this was no longer being used and the `Initializer` property was removed from the `Parameter` type, I've removed this api. We should now be adding initialization statements when constructing a method body.